modules/RTC/JitsiLocalTrack.js

```diff
@@ -74,6 +74,13 @@ export default class JitsiLocalTrack extends JitsiTrack {
         // track resolution.
         this.resolution = RTCBrowserType.isFirefox() ? null : resolution;
 
+        if (RTCBrowserType.usesNewGumFlow()) {
```

Comment:
why not combine this with the above check in some if-elseif-else ?

Reply:
Essentially I was having fun and trying to reduce the amount of red in the changeset. My needless desire to reduce red was because the previous attempt at using adapter applied adapter globally, causing a scramble situation to fix all environments. That spurred me to avoid changing existing code as little as possible to be iterative in applying adapter. I'll combine the checks.